pages/draw.md

```diff
@@ -630,11 +630,12 @@ The default text style behavior is adjusted to account for the parent point:
     - The `anchor` of the `text` controls the text's placement *relative to the size and position* of its parent point.
     - The `anchor` of the `points` portion moves the *entire entity* (point + text) relative to the underlying geometry point.
 - **Collision**:
-  - The point is required, but its text is optional: while the `points` portion of the style will render according to its collision test, the `text` portion will only render if **both** it and its parent point passed collision tests, e.g. if the point is occluded, then the text won't render either, even if it is not occluded.
+  - The point is required, but its text is not: while the `points` portion of the style will render according to its collision test, the `text` portion will only render if **both** it and its parent point passed collision tests, e.g. if the point is occluded, then the text won't render either, even if it is not occluded.
   - Different collision behaviors can be achieved by setting the `collide: false` flag on either or both of the point and text:
     - Both `collide: true` (default): nothing will overlap, text will only be rendered if point also fits.
-    - Points `collide: false`, text `collide: true`: all points will render, text will render if it fits.
-    - Points `collide: true`, text `collide: false`: points will render if they fit, in which case their attached text will also render, even if it overlaps something else.
+    - Points `collide: false`, text `collide: true` (default) and text `optional: true`: all points will render, text will render over points when the text fits (text will collide text but not points). This setting is helpful for labeling a dot density map.
+    - Points `collide: false`, text `collide: true` (default): only points with text that fits will render
```

Comment:
The earlier copy said (I think it was wrong): 

`    - Points `collide: false`, text `collide: true`: all points will render, text will render if it fits.`